project_task_project_mandatory/README.rst

```diff
@@ -0,0 +1,48 @@
+Enhancements related to Project Management.
+
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Project Task Project Mandatory
+==============================
+
+This module extends the functionality of project to support to make project required at task.
+
+Installation
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (project_timesheet, project_issue_sheet)
```

Comment:
Why does this module need inherit two modules: project_timesheet, project_issue_sheet?